bin/pasta_show_maintainers.py

```diff
@@ -193,42 +204,35 @@ def show_maintainers(config, sub, argv):
     parser.add_argument('--smallstat', action='store_true', help='Simple view')
     parser.add_argument('--bytes', action='store_true', help='Show byte counts')
     parser.add_argument('--outfile', type=str, help='Output to a csv file')
-    parser.add_argument('--file', type=str, help='Specify MAINTAINERS file')
     parser.add_argument('--filter', type=str, help='Filter by file list')
-    parser.add_argument('--commit', type=str, 
+    parser.add_argument('--revision', type=str, 
                             help='Specify a commit hash for Linux repo')
-    parser.add_argument('--cpu', dest='cpu_factor', metavar='cpu', type=float,
-                        default=1.0, help='CPU factor for parallelisation '
-                                          '(default: %(default)s)')
+
     args = parser.parse_args()
-    d_repo = config.repo_location
-    repo = pygit2.Repository(d_repo)
-    commit_version = args.commit if args.commit else 'HEAD'
-    commit = repo.revparse_single(commit_version)
-    tree = commit.tree
-        
+
     if args.filter:
         filter_by_files = file_to_string(args.filter).splitlines()
         # Automatically find kernel version from file name "v5.6"
-        version_regex = re.compile(r'v[0-9]+.[0-9]+')
-        kernel_version = version_regex.findall(args.filter)[0]
+        extract_version = '.'.join(args.filter.split('.')[1:3])
```

Comment:
Yikes, I just realised that dot is probably not the best choice as separator :-)

Reply:
Let me try another approach, will get back to you.

Reply:
That was just my work in progress, sorry about the mess :) 
Should I go back to regex?

Reply:
No worries. I'm not sure let. I'm playing around with some possibilities. Will provide you a patch for on top of your series.